components/pedigree/resources/src/main/resources/pedigree/editor.css

```diff
@@ -606,6 +606,28 @@
   width: 48%;
   float: right;
 }
+.menu-box .field-deceasedAge {
+  width: 47%;
+  float: left;
+}
+.menu-box .field-deceasedCause {
+  width: 48%;
+  float: right;
+}
+.menu-box .field-deceasedAge {
+  padding-top: 0;
+  padding-bottom: 0.8em;
+}
+.menu-box .field-deceasedCause {
+  padding-top: 0;
+  padding-bottom: 0.8em;
+}
```

Comment:
I'd like to know your reasons @allasm . Again this is nothing urgent, but I think we should at some point come up with some best practices we can all agree on, so that we can avoid such discussions at review time.

@veronikaslc you don't need to change anything here for now.